### PR TITLE
[MM-13279] Add automated test using cypress for Account Settings > Add Nickname

### PR DESCRIPTION
**Pull request by patniharshit**
#### Summary
 
     - Click on hamburger menu
     - Select Account Settings > General > Nickname > Edit
     - Type in a Nickname and press "Save"
     - Close Account Settings modal
     - In a channel you belong to, click the channel name and select Manage Members (or View Members)
     - In the "Search Users" field, type your username

Expected:

    - After 3) the Nickname you saved is displayed in the Nickname field
    - After 6) your nickname is displayed in parentheses next to your name

#### Ticket Link
https://github.com/mattermost/mattermost-server/issues/10044

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Added or updated unit tests (required for all new features)

### CODE REVIEW

File: cypress/integration/account_settings/general/nickname.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        cy.toAccountSettingsModal('user-1');
+        cy.get('#accountSettingsHeader > .close').click();
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        cy.get('#manageMembers').should('be.visible');
+        cy.get('#manageMembers').click();
+
+        // 5. Search for username and check that expected nickname is present
+        cy.get('.modal-title').should('be.visible');
+        cy.get('input[placeholder="Search users"]').should('be.visible').type('Victor Welch');
```
Comment (saturninoabril) [inline cypress/integration/account_settings/general/nickname.js]: Same here, please use `searchUsersInput` ID once added.

**Comment by hanzei** [comment]
Hey @patniharshit,

I appreciate your contribution. :+1: Do you have questions about the provided feedback? Do you have any other questions?

**Comment by hanzei** [comment]
@patniharshit texted me and said, that we still take care of finishing this one.

**Comment by saturninoabril** [comment]
Thanks @hanzei for the update.
I went ahead and did minor update and rebase.  I hope it's just fine @patniharshit.
@hmhealey This should be ready for dev review.

**Comment by patniharshit** [comment]
@saturninoabril no issues. Thanks for completing it.

**Pull request merged**